plugin/evm/vm_test.go

```diff
@@ -124,6 +124,69 @@ func init() {
 	}
 }
 
+func newPrefundedGenesis(
+	balance int,
+	addresses ...common.Address,
+) *core.Genesis {
+	alloc := core.GenesisAlloc{}
+	for _, address := range addresses {
+		alloc[address] = core.GenesisAccount{
+			Balance: big.NewInt(int64(balance)),
+		}
+	}
+
+	genesis := newGenesis()
+	genesis.Alloc = alloc
+
+	return genesis
+}
+
+func newGenesis() *core.Genesis {
```

Comment:
I feel like the way we're testing the VM by passing in a string-ed json is confusing.... added some code to just pass around a genesis struct instead + initialize a funded genesis instead of imports w/ shared memory.